### ISSUE TITLE
[Suggestion] "alert" and "callout"

### ISSUE THREAD

**Issue by Adysone** [almasaeed2010/AdminLTE#220]
Hello,

It's not really an issue, but more a suggestion. Actually, I would like to use standard BS3 "alert" without the icon in a circle and without the extra margin on the left of the box.

I think I can't be the only one. Would it be possible to add an extra class to "alert" like "alert-adminLTE" for these extra features and let the classic "alert" with standard classes?

```html
<!-- Standard alert -->
<div class="alert alert-success">
    <i class="fa fa-check"></i>
    <button type="button" class="close" data-dismiss="alert" aria-hidden="true">×</button>
    <strong>Alert!</strong> Success alert preview. This alert is dismissable.
</div>

<!-- AdminLTE alert -->
<div class="alert alert-adminLTE alert-danger">
    <i class="fa fa-ban"></i>
    <button type="button" class="close" data-dismiss="alert" aria-hidden="true">×</button>
    <strong>Alert!</strong> Danger alert preview. This alert is dismissable. A wonderful serenity has taken possession of my entire soul, like these sweet mornings of spring which I enjoy with my whole heart.
</div>
```
![image](https://cloud.githubusercontent.com/assets/6309463/5935205/8249bb22-a6d8-11e4-9c01-6480e77fb3c0.png)

The other point is "callout". I think "callout-success" is missing.

![image](https://cloud.githubusercontent.com/assets/6309463/5935297/985bf03c-a6d9-11e4-8420-de343b582282.png)

What do you think?

**Comment by almasaeed2010** [comment]
Hi,

I had remove the circle from AdminLTE 2.0 because it's not displaying properly on some phones. I got a few emails telling me about this. The solution had to be removing them and adding support for big icons in h4's to produce something like this:

![screen shot 2015-01-28 at 10 57 11 am](https://cloud.githubusercontent.com/assets/1512664/5941031/7672c8c6-a6dc-11e4-89d3-7580187c5f55.png)

If the .icon class was not added to the font icon, it would behave normally and not add any extra margin or padding.

**Comment by Adysone** [comment]
Hi,

This version 2.0 seems great!
But on your screenshots, I can see that alerts are not centered because of
the left padding.
The .icon class wasn't added on the font icon, but in the CSS, the CSS rule
impacts also .fa class.

Thank you for your answer.

**Comment by lenamtl** [comment]
Hi, I was not a fan of the alert icon circle, I think this was the first modification I did.

**Comment by almasaeed2010** [comment]
Opps! I forgot to remove the original margin-left from alerts. I fixed it now. You can view the changes here http://almsaeedstudio.com/themes/AdminLTE/pages/UI/general.html

**Comment by Adysone** [comment]
That's really nice. Will be the 2.0 fully compatible with the 1.3.1?

-- 
Anthony D.

**Comment by almasaeed2010** [comment]
A few modifications to the main layout will be required such as the .wrapper must wrap the whole site after the body tag instead of only the content and sidebar. So really just one line to replace. I did my best to not change anything else. I added as much fallback fixes as I could. Once the alpha release is out you guys will be able to test it and let me know if there are thing that I haven't noticed.
